app/src/main/AndroidManifest.xml

```diff
@@ -151,4 +149,17 @@
             android:windowSoftInputMode="adjustNothing" />
     </application>
 
```

Comment:
We do need `QUERY_ALL_PACKAGES` since Autofill has to compute the certificate hash for every app it is triggered in. Google may end up adding an exception for this use case, but for now this permission is required.

Reply:
Interesting, Android Studio was very insistent about having to remove it. I'll add it back when I next work on this.